docs/reference/commandline/system_prune.md

```diff
@@ -0,0 +1,70 @@
+<!--[metadata]>
++++
+title = "system prune"
+description = "Remove unused data"
+keywords = [system, prune, delete, remove]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# system prune
+
+```markdown
+Usage:	docker system prune COMMAND
+
+Delete unused data
+
+Options:
+  -a, --all     Remove all unused images not just dangling ones
+  -f, --force   Do not prompt for confirmation
```

Comment:
(Wondering) do we expect a `--format` option to be added here; if so, `-y` ("yes") could be an alternative